public-api/src/main/resources/public-api.yaml

```diff
@@ -1468,6 +1506,59 @@ definitions:
         items:
           $ref: "#/definitions/Variant"
 
+  GenomicFilterOption:
```

Comment:
This is the json structure used to send the filter options back. Checked would be true initially and when user unchecks and sends it back to api call the results would be eliminated.

Reply:
option is the actual gene/ clinical significance/ consequence value. count is the result count for that option. min and max are only used for allele count / allele number / allele frequency.